site/third-party/bundlers.md

```diff
@@ -41,3 +41,7 @@ with your provider on what the correct logic is.
 It might be the case that you want to use a different RPC provider for your bundler traffic and your node traffic. This is a common use case, and you can do this by leveraging the [`split`](/packages/aa-core/split-transport) transport and passing it to your `createSmartAccountClient` call. For example:
 
 <<< @/snippets/aa-core/splitTransport.ts
+
+## Zora and Fraxtal
+
+Using a split Bundler and Node RPC setup is required for Fraxtal, Fraxtal Testnet, Zora, and Zora Sepolia networks since Alchemy currently only supports Account Abstraction endpoints for those networks.
```

Comment:
do we have any suggested 3rd party providers for RPC for theses chains? might be helpful to link out to them here

Reply:
Good idea, added links